assets/src/components/mapPage.tsx

```diff
@@ -208,6 +216,9 @@ const MapPage = (): ReactElement<HTMLDivElement> => {
   const [{ searchPageState, openView }, dispatch] =
       useContext(StateDispatchContext),
     { selectedEntity = null } = searchPageState
+  const [selectedVehicleOrGhost, setSelectedVehicleOrGhost] = useState<
```

Comment:
nit(non-blocking): I was a little confused by this name at first. If you have any ideas on how to make it clearer that this controls the panel vehicle I think that'd be good, but it's otherwise fine as is.

Reply:
Oh, good point, I literally just used the same name that the application-wide state reducer does but that makes much less sense here. How about `selectedRightPanelVehicleOrGhost`?

Reply:
Sounds good 👍🏻

Reply:
e93f421b